fairlearn/reductions/grid_search/grid_search.py

```diff
@@ -74,6 +76,10 @@ def accumulate_integer_grid(self, index, max_val):
                 self.accumulate_integer_grid(index+1, max_val-abs(current_value))
 
 
+_constraint_strings = {"demographic_parity": DemographicParity(),
```

Comment:
We should standardise the strings. @romanlutz  has used CamelCase, but I think this is more like `sklearn` usage. For example:
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html#sklearn.metrics.mean_squared_error

Reply:
It should definitely be the same, not two different ways of specifying it.
Let's also fix the name in a separate variable 
```
_demographic_parity = "demographic_parity"
```
so that it can be used in tests without having to make 100 replacements

Reply:
Done - can you take a look and tell me if you like it :-)